src/metabase/api/pulse.clj

```diff
@@ -54,16 +56,18 @@
 
 (defendpoint PUT "/:id"
   "Update a `Pulse` with ID."
-  [id :as {{:keys [name cards channels]} :body}]
+  [id :as {{:keys [name cards channels skip]} :body}]
```

Comment:
That would appear in the JSON object, so I think we should prefer names that are valid JS identifiers, i.e. `skip_if_empty`